mds/authent/migrations/0003_auto_20190903_1529.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.11 on 2019-09-03 15:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('authent', '0002_auto_20190124_0928'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='application',
+            name='owner',
+            field=models.UUIDField(blank=True, db_index=True, help_text='Unique identifier for the owner of the application', null=True),
+        ),
+    ]
```

Comment:
You got this migration because you typed `./manage.py migrate` with no argument, which is bad practice.

Always type the app you want to migrate as the first argument. You shouldn't have any migration in authent, because you didn't alter it.

Reply:
Merci pour l'ifno